mars/storage/ray.py

```diff
@@ -132,10 +147,26 @@ async def get(self, object_id, **kwargs) -> object:
 
     @implements(StorageBackend.put)
     async def put(self, obj, importance=0) -> ObjectInfo:
-        object_id = ray.put(obj)
+        if support_specify_owner():
+            ray_supervisor_actor = await self._get_ray_supervisor_actor()
+            object_id = ray.put(obj, _owner=ray_supervisor_actor)
+        else:
+            object_id = ray.put(obj)
         # We can't get the serialized bytes length from ray.put
         return ObjectInfo(object_id=object_id)
 
+    async def _get_ray_supervisor_actor(self):
```

Comment:
In our design, the storage lib cannot be aware of the existence of services. Is it possible to integrate this logic into storage service?

Reply:
@qinxuye @hekaisheng Do you have any suggestions about how to pass supervisor from service to ray storage backend?